### PR TITLE
Include appveyor yml

### PR DESCRIPTION
**Pull request by mglodack**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 1.0.{build}
```
Comment (awseward) [inline appveyor.yml]: Care to clarify this bit?

**Comment by awseward** [comment]
Thanks for getting on this @mglodack!

https://github.com/awseward/Bugsnag.NET/issues/10

File: appveyor.yml (hunk 1 above)
Comment (mglodack) [inline appveyor.yml]: Sets the build version number. This was exported from appveyor, and could probably change.

Comment (awseward) [inline appveyor.yml]: If that's all it does, that's fine.

**Comment by awseward** [comment]
+1

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Build status](https://ci.appveyor.com/api/projects/status/86j23g6eltupv6ox?svg=true)](https://ci.appveyor.com/project/mglodack/bugsnag-net)
+
```
Comment (awseward) [inline README.md]: Could we put this linline w/ the title?

**Comment by awseward** [comment]
Looks great! Thanks, @mglodack!

**Pull request merged**